ChemistryLib/PhreeqcIOData/KineticReactant.cpp

```diff
@@ -13,28 +13,26 @@
 
 namespace ChemistryLib
 {
-std::ostream& operator<<(std::ostream& os,
-                         KineticReactant const& kinetic_reactant)
+void KineticReactant::print(std::ostream& os,
```

Comment:
Just curious, why change the output operator to `print`?

Reply:
Because the data type of `KineticReactant.amount` is changed to `MeshLib::PropertyVector<double>*`. `chemical system id`  is required from outside of the class `KineticReactant` to access elements by index when looping over the chemical systems. 

My preference is also to keep the operator overloading function herein. Can we keep it from drop-in replacement?